spec/datetimeformat.html

```diff
@@ -592,7 +600,11 @@
       </p>
 
       <p>
-        The function returns a new object whose properties and attributes are set as if constructed by an object literal assigning to each of the following properties the value of the corresponding internal slot of this DateTimeFormat object (see <emu-xref href="#sec-properties-of-intl-datetimeformat-instances"></emu-xref>): locale, calendar, numberingSystem, timeZone, hour12, weekday, era, year, month, day, hour, minute, second, and timeZoneName. Properties whose corresponding internal slots have the value *undefined* are not assigned.
+        The function returns a new object whose properties and attributes are set as if constructed by an object literal assigning to each of the following properties the value of the corresponding internal slot of this DateTimeFormat object (see <emu-xref href="#sec-properties-of-intl-datetimeformat-instances"></emu-xref>): locale, calendar, numberingSystem, timeZone, hourCycle, weekday, era, year, month, day, hour, minute, second, and timeZoneName. Properties whose corresponding internal slots have the value *undefined* are not assigned.
+      </p>
+
+      <p>
+        For web compatibility reasons, if the property hourCycle is set to values *h12* or *h24*, the corresponding property hour12 should be set as well.
```

Comment:
This works, but I'm not a huge fan of this wording for a couple minor reasons:
- Seems like, suddenly, `hourCycle` comes last in the enumeration order of the resolvedOptions object. This is extremely minor, but it's a rather unmotivated (only by spec wording constraints) observable effect of the change.
- Comments like "for web compatibility reasons" belong in notes, rather than normative text (otherwise it looks a little bit like an "Annex B" normative-optional thing). There are tons and tons of things in all kinds of specs are for web compatibility reasons; probably better to just say what the behavior is.

It would be best to reword the resolvedOptions methods in a more imperative way, as suggested in https://github.com/tc39/ecma402/issues/124 .